lib/ios/RNNModalManager.m

```diff
@@ -33,6 +33,13 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
     return self;
 }
 
+- (BOOL)canAnimate:(UIViewController *)viewController {
+    // formSheet and pageSheet modals have a custom slide-from-bottom animation. Applying any custom
+    // animation here will break that default iOS animation.
+    return viewController.modalPresentationStyle != UIModalPresentationFormSheet &&
```

Comment:
@yogevbd We should check `modalPresentationStyle` from resolvedOptions, right?

Reply:
also, `modalPresentationStyle` is a Text object and `UIModalPresentationFormSheet` is an integer, best thing is to add a new option object type for `modalPresentationStyle`. I can take it

Reply:
actually we apply `modalPresentationStyle` on the top parent and we also present/hide the top parent so it looks good to me